terraform/data.tf

```diff
@@ -0,0 +1,14 @@
+data azurerm_key_vault vault {
+  name                = var.key_vault_name           #keyvault name
```

Comment:
The comments are not particularly useful in this file

Reply:
done...